StreemKernel/0.3.0-swift55/StreemKernel.podspec.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "StreemKernel",
+  "version": "0.3.0-swift55",
+  "summary": "Streem Kernel Cocoapod",
+  "description": "StreemKernel is for use by Streem's iOS SDKs only.",
+  "homepage": "https://streem.pro",
+  "platforms": {
+    "ios": "13.0"
```

Comment:
I don't understand why this isn't set to `14.0` since we did push that change? Same with the swift versions.

Reply:
must have not been set in `StreemKernel`s `-public-spec`. Let's get that in there

Reply:
Oh I see, thanks.

Reply:
You guys are changing our minimum supported iOS version to 14.0???

Reply:
No this is only for StreemGuidedScanKit.

Reply:
🤔